pkg/controller/migcluster/migcluster_controller.go

```diff
@@ -207,14 +204,12 @@ func (r *ReconcileMigCluster) setupRemoteWatch(cluster *migapi.MigCluster) error
 	if cluster.Spec.IsHostCluster {
 		restCfg, err = config.GetConfig()
 		if err != nil {
-			log.Trace(err)
-			return err
+			return liberr.Wrap(err)
 		}
 	} else {
 		restCfg, err = cluster.BuildRestConfig(r.Client)
 		if err != nil {
-			log.Trace(err)
-			return err
+			return liberr.Wrap(err)
```

Comment:
@jortel is this generally in line with what you were expecting from this PR? A switch from `log.Trace()` over to `liberr.Wrap()`?

Reply:
Yes, exactly.